resource/plot.ui

```diff
@@ -2,14 +2,13 @@
 <ui version="4.0">
  <class>DataPlotWidget</class>
  <widget class="QWidget" name="DataPlotWidget">
-  <property name="geometry">
```

Comment:
Are these mutually exclusive? Wouldn't you want a suggested size and a minimum size?

Reply:
This doesn't seem to have any effect which is why I replaced it with a minimum